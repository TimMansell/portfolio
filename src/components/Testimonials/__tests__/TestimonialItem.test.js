import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

import TestimonialsItem from '../TestimonialItem';

const props = {
  description: 'Description',
  author: 'Author',
  company: 'Company',
  position: 'Position',
};

describe('TestimonialsItem', () => {
  it('should render my component', () => {
    render(<TestimonialsItem {...props} />);
  });

  it('should match snapshot', () => {
    const { asFragment } = render(<TestimonialsItem {...props} />);

    expect(asFragment()).toMatchSnapshot();
  });
});
