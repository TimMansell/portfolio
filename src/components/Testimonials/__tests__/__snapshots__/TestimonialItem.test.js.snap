// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestimonialsItem should match snapshot 1`] = `
<DocumentFragment>
  <p
    class="text"
  >
    Description
  </p>
  <p
    class="name"
  >
    Author (Position)
  </p>
  <p
    class="company"
  >
    Company
  </p>
</DocumentFragment>
`;
